src/Hosting/Hosting/src/Host.cs

```diff
@@ -70,9 +70,11 @@ public static IHostBuilder CreateDefaultBuilder(string[] args)
             builder.ConfigureAppConfiguration((hostingContext, config) =>
             {
                 var env = hostingContext.HostingEnvironment;
+                
+                var reloadOnChange = hostingContext.Configuration.GetValue("hostbuilder:configreload", true);
```

Comment:
```suggestion
                var reloadOnChange = hostingContext.Configuration.GetValue("hostbuilder:configreload", defaultValue: true);
```

Reply:
Maybe change the name a little too? Descriptive is better than terse IMO. `hostBuilder:reloadConfigOnChange` ?

Reply:
I'll take care of that and get this merged.